_data-prepper/logs.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Log4j configuration
+nav_order: 12
+---
+
+# Log4j configuration
+
+This section provides the details for configuring Log4j. 
+
+## Logging 
+
+The following describes how Data Prepper performs logging. Data Prepper uses [SLF4J](http://www.slf4j.org/) with a [Log4j 2 binding](http://logging.apache.org/log4j/2.x/log4j-slf4j-impl/). 
+
+<!--- Editorial: Is it safe to remove this first introductory sentence? I'm not sure if it's necessary or helpful to users.--->
+
+For Data Prepper version 2.0 and later, the Log4j 2 configuration file can be found and edited in `config/log4j2.properties` in the application's home directory. The default properties for Log4j 2 can be found in `log4j2-rolling.properties` in the *shared-config* directory.
+
+For Data Prepper versions before 2.0, Log4j 2 configuration can be overridden by setting "log4j.configurationFile" system property when running Data Prepper. Default properties for Log4j 2 can be found in `log4j2.properties` in the *shared-config* directory. 
```

Comment:
```suggestion
For Data Prepper versions before 2.0, the Log4j 2 configuration can be overridden by setting the "log4j.configurationFile" system property when running Data Prepper. Default properties for Log4j 2 can be found in `log4j2.properties` in the *shared-config* directory. 
```

Reply:
Delete the second sentence, as it appears in the preceding paragraph. However, note that the text in code format is different in each case.